platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -31,6 +31,8 @@
 #define _ARM_CRYPTO_MAX_CIPHER_KEY_LENGTH      64
 #define _ARM_CRYPTO_MAX_AUTH_KEY_LENGTH        64
 #define _ARM_CRYPTO_MAX_IV_LENGTH              16
+#define _ARM_CRYPTO_MAX_DATA_LENGTH	       65536
```

Comment:
There is a somewhat hidden assumption in the code that this is divisible by 16. Otherwise the aes_gcm operations may write beyond the end of the temporary buffer. There should be a clear comment about it (not just "must be divisible by 16" because that leaves the reader wondering why), or 15 bytes of extra room in the array.

Reply:
I will add a comment above. Please review it and let me know if it is explanation is sufficient

Reply:
The comment above is still a work-in-progress. Please leave your suggestions on how to improve it further. Thanks!